libvirt.go

```diff
@@ -204,6 +255,58 @@ func (l *Libvirt) Events(dom string) (<-chan DomainEvent, error) {
 	return c, nil
 }
 
+// Migrate synchronously performs a domain migration to the hypervisor
+// specified by dest. The flags argument determines the type of migration
```

Comment:
I feel we should specify that `dom` is the name of the domain and that `dest` should be a URL probably of the form `qemu+tcp://` or `qemu+ssh://`, etc.  Leave examples.